app/build.gradle

```
@@ -13,11 +13,11 @@ android {
             storePassword keystoreProperties['storePassword']
         }
     }
-    compileSdkVersion 31
+    compileSdkVersion 34
     defaultConfig {
         applicationId "com.talkable.demo"
-        minSdkVersion 16
-        targetSdkVersion 31
+        minSdkVersion 19
+        targetSdkVersion 34
         versionCode 34
         versionName "0.5.3"
```

you should also update `versionCode` and `versionName`
```suggestion
        versionCode 35
        versionName "0.5.4"
```